git_repo/services/service.py

```diff
@@ -585,6 +614,9 @@ def request_fetch(self, user, repo, request, pull=False, force=False): #pragma:
     def request_create(self, user, repo, from_branch, onto_branch, title, description=None, auto_slug=False):
         raise NotImplementedError
 
+    def review(self):
+        raise NotImplementedError
```

Comment:
so, what's your intention with that?

I assumed you were about to do some sort of `git <target> request <project> review` command.

Reply:
Sorry, I think I forgot to remove this function. Right, at first we wanted to have something like `git <target> request <project> review`, but then we decided to have `git <target> request fetch <change-id>` instead.